File: Leetcode/118.pascal-sTriangle/119.py
class Solution(object):
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        l=[0]*(rowIndex+1)
        l[0]=1
        for i in range(1,rowIndex+1):
        	for j in range(i,0,-1):
        		l[j]+=l[j-1]
        return l;

File: Leetcode/118.pascal-sTriangle/119.c
int* getRow(int rowIndex, int* returnSize) {
    int *array = (int*)malloc((rowIndex+1)*sizeof(int));
    //memset(array, 0, sizeof(int) * (rowIndex + 1));
    for (int i = 0; i < rowIndex + 1; ++i)
    array[i] = 1;
    for(int i = 2;i<=rowIndex;i++)
    	for(int j = i - 1;j>0;j--)
    		array[j]+= array[j-1];
    *returnSize = rowIndex+1;
    return array;
}
